### DISCUSSION TITLE
Live point and view like Android version but with Win/Mac using DSLR with GPS?

### DISCUSSION THREAD

**Discussion by P2AK** [Stellarium/stellarium#1642] [General]
Is it possible to get the same live point and view effect that exists on the Android/cell phone version of Stellarium on a Mac or Windows computer using a USB attached DSLR that has GPS in it?  My camera is a Canon EOS 7DMKII and it has GPS and can operate like a compass so I *think* it has an accelerometer in it?  Is it possible to point the camera to different spots in the sky and see it change the view on the computer screen?

**Comment by gzotti** [comment]
Currently there is no such function. If you write a plugin or communication software using the RemoteControl API, it is possible.

**Comment by alex-w** [comment]
Do you want to see AR mode for desktop edition of planetarium?

**Reply by P2AK** [reply]
If I understand what AR is, then yes, that would be really nice.  I'd love to be able to use my DSLR's GPS or to point it correctly at wide field or even deep space objects.  I'm just not a programmer and unfortunately I don't have the skill set needed to write a plugin.
